service/pom.xml

```diff
@@ -310,6 +312,15 @@
                         </execution>
                     </executions>
                 </plugin>
+                <plugin>
+                    <artifactId>maven-release-plugin</artifactId>
+                    <version>3.0.0-M1</version>
+                    <configuration>
+                        <tagNameFormat>v@{project.version}</tagNameFormat>
+                        <arguments>-Dmaven.test.skip=true -Dcheckstyle.skip -Dpmd.skip -Dcpd.skip -Dfindbugs.skip -Dspotbugs.skip</arguments>
+                        <scmCommentPrefix>[ci skip]</scmCommentPrefix>
```

Comment:
when maven updates pom files as part of the release we do not want to run the `Build` workflow; This config adds a `[ci skip]` message in the commit that maven release pushes, and since it has `[ci skip]` github will not tirgger any workflow.
![image](https://github.com/SolaceLabs/event-management-agent/assets/59615308/630f207d-dec5-4ab5-a128-38a748d95999)